CONTRIBUTING.md

```diff
@@ -45,32 +45,30 @@ git checkout -b 25-customize_overlays
 ### Commit in the Pull Request
 There is no convention for the commit message in the Pull Request.
 The most important part is the title of the Pull Request, because:
-- Everyone must use Pull Request, no direct commit allowed on main branch
-- The commits of a Pull Request are almost always squashed
-- The title of the Pull Request is used as proposal for the maintainer merging the Pull Request
+- Everyone must use Pull Request, no direct commit allowed on the `main` branch.
+- The commits of a Pull Request are almost always squashed.
+- The title of the Pull Request is used as proposal for the maintainer merging the Pull Request.
 
 ### Open a Pull Request
 
-At this point, you should switch back to your main branch and make sure it's up-to-date with `bpmn-visualization-R`
-`main` branch:
+At this point, you should switch back to your own branch and make sure it's up-to-date with the `main` branch of the `bpmn-visualization-R` repository:
 
 ```sh
 git remote add upstream git@github.com:process-analytics/bpmn-visualization-R.git
 git checkout main
 git pull upstream main
 ```
 
-Then update your feature branch from your local copy of main, and push it!
+Then update your feature branch from your local copy of the `main` branch, and push it!
 
 ```sh
 git checkout 25-customize_overlays
 git rebase main
 git push --set-upstream origin 25-customize_overlays
 ```
 
-Finally, go to GitHub and [make a Pull Request](https://help.github.com/articles/creating-a-pull-request) with labels :smile:
+Finally, go to GitHub and [make a Pull Request](https://help.github.com/articles/creating-a-pull-request) with labels :smile:  
```

Comment:
❓ can we use directly emoji instead of GH syntax? This would increase portability

Reply:
```suggestion
Finally, go to GitHub and [make a Pull Request](https://help.github.com/articles/creating-a-pull-request) with labels 😄   
```